proposals/p2240.md

```diff
@@ -136,7 +138,7 @@ fn SumInts(... each param: i64) -> i64 {
 
 ```carbon
 // Concatenates its arguments, which are all convertible to String
-fn StrCat[... each T:! ConvertibleToString](... each param: each T) -> String {
+fn StrCat(... each param: some ConvertibleToString) -> String {
```

Comment:
Why this change? It seems like it would not be desirable to use `some` which is not in any accepted proposal.